Cargo.toml

```diff
@@ -3,11 +3,18 @@ name = "pg_query"
 description = "PostgreSQL parser that uses the actual PostgreSQL server source to parse SQL queries and return the internal PostgreSQL parse tree."
 version = "6.0.0"
 edition = "2021"
-documentation = "https://docs.rs/pg_query/"
-build = "build.rs"
+documentation = "https://docs.rs/pg_query"
 license = "MIT"
-readme = "./README.md"
 repository = "https://github.com/pganalyze/pg_query.rs"
+include = [
+    # pg_query.rs
+    "README.md", "build.rs", "src/**/*.rs",
+    # libpg_query
+    "Makefile", "pg_query.h",
+    "libpg_query/{src,vendor}/**/*.{c,h}",
+    "libpg_query/protobuf/{pg_query.pb-c.*,pg_query.proto}",
+    "!libpg_query/src/postgres/*.c",
```

Comment:
There is an `exclude` field, but it can't be used in conjunction with `include`. Instead, exclusion paths have to use `!` https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields

Reply:
Can you explain why its okay to exclude those source files? (wouldn't they be needed to build?)

Reply:
I just guessed that files not in the `include` folder weren't actually needed.

Double checking that, and it seems to be true:
```sh
rm libpg_query/src/postgres/*.c
cargo clean
cargo build

Reply:
Hmm `cargo test` seems to fail with a linking error. It's surprising that `cargo build` doesn't catch that.

I'll remove this line.

Reply:
I think practically that only affects these four files:

```
libpg_query/src/postgres/include/copyfuncs.funcs.c
libpg_query/src/postgres/include/copyfuncs.switch.c
libpg_query/src/postgres/include/equalfuncs.funcs.c
libpg_query/src/postgres/include/equalfuncs.switch.c
```

And yes, it actually does appear correct that they are not needed (we could probably figure out a way to skip over that when extracting the code in libpg_query). Maybe just to avoid future breakage we could make the rule something like:

```
!libpg_query/src/postgres/include/{copyfuncs,equalfuncs}.{funcs,switch}.c
```

If that works?

Reply:
Ah, looks like I sent my comment before you commented - in that case disregard what I said :)

Reply:
Adding back these files just increased the package size from 1.8 MB to 2.4 MB, so it's not a huge difference. Maybe in the future we should evaluate what files can be removed from the libpg_query repo instead of trying to filter them out here.